w5e1.py

```diff
@@ -0,0 +1,82 @@
+"w5e1" 
+import random
+import os
+
+# pokemon s1e1 -> Ash late to professor Oak so can't choose pokemon...
+from datetime import datetime, timedelta
+
+MONDAYS = 1
+
+
+def pokemon():
+    pokemons = ['Bulbasaur', 'Squirtle', 'Charmander']
+    starter_pokemon = random.choice(pokemons)
+    print(starter_pokemon)
```

Comment:
separate IO from logic   - to make the function reusable with different outputs

Reply:
I chose to leave the print anyway because the function of the function was to try the random.